DEV-9/DEV-9.Tests/MailRuLoginTests.cs

```diff
@@ -0,0 +1,71 @@
+﻿using System;
+using DEV_9.PageObjects.MailRu;
+using NUnit.Framework;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Chrome;
+using OpenQA.Selenium.Support.UI;
+
+namespace DEV_9.Tests
+{
+    // The task asks to test only one of the services.
+    [TestFixture]
+    public class MailRuLoginTests
+    {
+        private IWebDriver driver;
+
+        [SetUp]
+        public void StartBrowser()
+        {
+            this.driver = new ChromeDriver();
+            this.driver.Url = "https://mail.ru";
+        }
+
+        [Test]
+        public void EmptyFieldsTest()
+        {
+            string Login = string.Empty;
+            string Password = string.Empty;
+            MailRuHomePage LoginPage = new MailRuHomePage(this.driver);
+
+            LoginPage.Login(Login, Password);
+
+            new WebDriverWait(this.driver, TimeSpan.FromSeconds(10)).Until(x => LoginPage.LoginErrorMessage.Displayed);
+
+            Assert.True(LoginPage.LoginErrorMessage.Displayed);
+        }
+
+        [Test]
+        public void WrongFieldsTest()
+        {
+            string Login = "AbraKadabra79182";
+            string Password = "BlaBla91283";
+            MailRuHomePage LoginPage = new MailRuHomePage(this.driver);
+
+            LoginPage.Login(Login, Password);
+
+            new WebDriverWait(this.driver, TimeSpan.FromSeconds(10)).Until(x => LoginPage.LoginErrorMessage.Displayed);
+
+            Assert.True(LoginPage.LoginErrorMessage.Displayed);
```

Comment:
you mix defferent layers. It is needed to create a methods in PO to operate with elements